lib/connectors/registry.rb

```diff
@@ -24,10 +24,10 @@ def connector_class(name)
       @connectors[name]
     end
 
-    def connector(name, configuration)
+    def connector(name, configuration, job_description)
       klass = connector_class(name)
       if klass.present?
-        return klass.new(configuration: configuration)
+        return klass.new(configuration: configuration, job_description: job_description)
```

Comment:
I think this change breaks many places:
https://github.com/elastic/connectors-ruby/blob/87a9c8aad96b798bf7b45667cb3fac15fa0b0ccf/lib/app/console_app.rb#L170

https://github.com/elastic/connectors-ruby/blob/87a9c8aad96b798bf7b45667cb3fac15fa0b0ccf/lib/core/heartbeat.rb#L22